cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NewTwoWindingsTransformerConversion.java

```diff
@@ -0,0 +1,426 @@
+        convertedModel.end2.b = rc0.b2;
+        convertedModel.end2.ratedU = interpretedT2xModel.end2.ratedU;
+        convertedModel.end2.terminal = interpretedT2xModel.end2.terminal;
+        convertedModel.end2.phaseAngleClock = interpretedT2xModel.end2.phaseAngleClock;
+
+        return convertedModel;
+    }
+
+    private void setToIidm(ConvertedT2xModel convertedT2xModel) {
+        TwoWindingsTransformerAdder adder = substation().newTwoWindingsTransformer()
+            .setR(convertedT2xModel.r)
+            .setX(convertedT2xModel.x)
+            .setG(convertedT2xModel.end1.g + convertedT2xModel.end2.g)
+            .setB(convertedT2xModel.end1.b + convertedT2xModel.end2.b)
+            .setRatedU1(convertedT2xModel.end1.ratedU)
+            .setRatedU2(convertedT2xModel.end2.ratedU);
+        identify(adder);
+        connect(adder);
+        TwoWindingsTransformer tx = adder.add();
+        convertedTerminals(tx.getTerminal1(), tx.getTerminal2());
+
+        setToIidmRatioTapChanger(convertedT2xModel, tx);
+        setToIidmPhaseTapChanger(convertedT2xModel, tx);
+
+        setToIidmPhaseAngleClock(convertedT2xModel, tx);
+        setRegulatingControlContext(convertedT2xModel, tx);
+    }
+
+    private void setToIidmRatioTapChanger(ConvertedT2xModel convertedT2xModel, TwoWindingsTransformer tx) {
+        TapChangerConversion rtc = convertedT2xModel.end1.ratioTapChanger;
+        if (rtc == null) {
+            return;
+        }
+
+        RatioTapChangerAdder rtca = newRatioTapChanger(tx);
+        setToIidmRatioTapChanger(rtc, rtca);
+    }
+
+    private void setToIidmPhaseTapChanger(ConvertedT2xModel convertedT2xModel, TwoWindingsTransformer tx) {
+        TapChangerConversion ptc = convertedT2xModel.end1.phaseTapChanger;
+        if (ptc == null) {
+            return;
+        }
+
+        PhaseTapChangerAdder ptca = newPhaseTapChanger(tx);
+        setToIidmPhaseTapChanger(ptc, ptca);
+    }
+
+    protected RatioTapChangerAdder newRatioTapChanger(TwoWindingsTransformer tx) {
+        return tx.newRatioTapChanger();
+    }
+
+    protected PhaseTapChangerAdder newPhaseTapChanger(TwoWindingsTransformer tx) {
+        return tx.newPhaseTapChanger();
+    }
+
+    private void setToIidmPhaseAngleClock(ConvertedT2xModel convertedT2xModel, TwoWindingsTransformer tx) {
+        // add phaseAngleClock as an extension, cgmes does not allow pac at end1
+        if (convertedT2xModel.end1.phaseAngleClock != 0) {
+            String reason = "Unsupported modelling: twoWindingsTransformer with phaseAngleClock at end1";
+            ignored("phaseAngleClock end1", reason);
+        }
+        if (convertedT2xModel.end2.phaseAngleClock != 0) {
+            TwoWindingsTransformerPhaseAngleClock phaseAngleClock = new TwoWindingsTransformerPhaseAngleClock(tx,
+                convertedT2xModel.end2.phaseAngleClock);
+            tx.addExtension(TwoWindingsTransformerPhaseAngleClock.class, phaseAngleClock);
+        }
+    }
```

Comment:
As being said above, it may be better as post-processing

Reply:
Agree to manage `PhaseAngleClock` as post-processing. All conversion code associated with `PhaseAngleClock` has been deleted. A new PR will be done as post-processing